wit/types.wit

```diff
@@ -6,16 +6,10 @@ interface types {
     
```

Comment:
For some reason, the addition of the `message` resource isn't showing as a change on this PR (while still being on this file), but I do agree we should keep it.

Reply:
Yeah that was in a previous commit. Definitely keeping as a resource for flexibility